merlin/dataloader/loader_base.py

```diff
@@ -544,40 +491,22 @@ def _create_tensors(self, gdf):
                         leaves, nest_offsets = pull_apart_list(leaves, device=self.device)
                         col_offsets = nest_offsets.iloc[col_offsets[:]]
 
-                    offsets[column_name] = col_offsets.reset_index(drop=True)
-                    list_tensors[column_name] = self._to_tensor(leaves)
-                x = x, list_tensors
-            tensors.append(x)
-            tensor_names.append(column_names)
-
-        if not offsets.empty:
-            offsets_tensor = self._to_tensor(offsets)
-            if len(offsets_tensor.shape) == 1:
-                offsets_tensor = offsets_tensor[:, None]
-            tensors.append(offsets_tensor)
-        del gdf, offsets
+                    tensors_by_name[column_name] = self._to_tensor(leaves), self._to_tensor(
+                        col_offsets
+                    )
```

Comment:
Super nit: I think black formatting makes this harder to read (at least for me). I wonder if using parens or `tuple()`, e.g., `tensors_by_name[column_name] = (self._to_tensor(leaves), self._to_tensor(col_offsets))`, improves readability with `black`.